src/pymovements/utils/checks.py

```diff
@@ -137,7 +142,9 @@ def check_is_mutual_exclusive(**kwargs) -> None:
         )
 
 
-def check_is_none_is_mutual(**kwargs) -> None:
+def check_is_none_is_mutual(
+        **kwargs: None | list[int] | np.ndarray,
```

Comment:
I would use `Any` here, as the type of the kwarg really does not matter

Reply:
done